src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/ConfirmTrainingProviderSteps.cs

```diff
@@ -0,0 +1,137 @@
+        public void GivenAConfirmTrainingProviderRequestStatingTheTrainingProviderIsCorrect()
+        {
+            endpoint = "TrainingProviderConfirmation";
+            TrainingProviderCorrect = true;
+            command = new ConfirmTrainingProviderRequest
+            {
+                TrainingProviderCorrect = true,
+            };
+        }
+
+        [Given("a ConfirmTrainingProviderRequest stating the training provider is incorrect")]
+        public void GivenAConfirmTrainingProviderRequestStatingTheTrainingProviderIsIncorrect()
+        {
+            endpoint = "TrainingProviderConfirmation";
+            TrainingProviderCorrect = false;
+            command = new ConfirmTrainingProviderRequest
+            {
+                TrainingProviderCorrect = false,
+            };
+        }
+
+        [Given("a ConfirmEmployerRequest stating the employer is correct")]
+        public void GivenAConfirmEmployerRequestStatingTheEmployerIsCorrect()
+        {
+            endpoint = "EmployerConfirmation";
+            EmployerCorrect = true;
+            command = new ConfirmEmployerRequest
+            {
+                EmployerCorrect = true,
+            };
+        }
+
+        [Given("a ConfirmEmployerRequest stating the employer is incorrect")]
+        public void GivenAConfirmEmployerRequestStatingTheEmployerIsIncorrect()
+        {
+            endpoint = "EmployerConfirmation";
+            EmployerCorrect = false;
+            command = new ConfirmEmployerRequest
+            {
+                EmployerCorrect = false,
+            };
+        }
+
+        [When("we send the confirmation")]
+        public async Task WhenWeSendTheConfirmation()
+        {
+            await _context.Api.Post(
+                $"apprentices/{_apprentice.Id}/apprenticeships/{_apprenticeship.Id}/{endpoint}",
+                command);
+        }
+
+        [Then("the response is OK")]
+        public void ThenTheResponseIsOK()
+        {
+            _context.Api.Response.EnsureSuccessStatusCode();
+        }
+
+        [Then("the response is BadRequest")]
+        public void ThenTheResponseIsBadRequest()
+        {
+            _context.Api.Response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+        }
+
+        [Then("the apprenticeship record is updated")]
+        public void ThenTheApprenticeshipRecordIsUpdated()
+        {
+            _context.DbContext.Apprenticeships.Should().ContainEquivalentOf(new
+            {
+                _apprenticeship.Id,
+                TrainingProviderCorrect,
+                EmployerCorrect,
+            });
+        }
+
+        [Then("the apprenticeship record remains unchanged")]
+        public void ThenTheApprenticeshipRecordRemainsUnchanged()
+        {
+            _context.DbContext.Apprenticeships
+                .Should().ContainEquivalentOf(_apprenticeship,
+                    compare => compare.Excluding(x => x.Apprentice));
+        }
+    }
+}
```

Comment:
Think we are missing the same tests when confirming an Employer

Reply:
The tests are inside the `ConfirmTrainingProvider` feature (because I forgot to rename it).  I've split them out into their own feature file now.